src/Classes/Dog.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+	namespace TopDog\Classes;
```

Comment:
please sort out the weird indentation in this file

Reply:
Done